modules/oe_theme_helper/oe_theme_helper.ckeditor5.yml

```diff
@@ -0,0 +1,50 @@
+oe_theme_helper_table_zebra_striping:
+  ckeditor5:
+    plugins:
+      - tableZebraStriping.TableZebraStriping
+    config:
+      table:
+        contentToolbar: [toggleTableZebraStriping]
+  drupal:
+    label: Table zebra striping
+    library: oe_theme_helper/internal.oe_theme_helper.ckeditor5.table_zebra_striping
+    elements:
+      - <table data-striped>
+    conditions:
+      plugins:
+        - ckeditor5_table
+      filter: filter_ecl_table
+
+oe_theme_helper_table_simple:
+  ckeditor5:
+    plugins:
+      - tableSimple.TableSimple
+    config:
+      table:
+        contentToolbar: [toggleTableSimple]
+  drupal:
+    label: Table simple
+    library: oe_theme_helper/internal.oe_theme_helper.ckeditor5.table_simple
+    elements:
+      - <table data-simple>
+    conditions:
+      plugins:
+        - ckeditor5_table
+      filter: filter_ecl_table
```

Comment:
For some reason, the condition doesn't work. It looks like conditions work with OR logic.

Reply:
I found out that you actually need a specific order of conditions to apply them correctly.
Whenever a "plugins" condition is present, the check bails out and doesn't evaluate the other condition types.
This because the plugins can be evaluated only when everything else has been evaluated.
But using the correct order (moving plugins always below) it will apply them correctly.